### PR TITLE
Fix links and versions reference in CSI Unity

### PR DESCRIPTION
**Pull request by panigs7**
# Description
This PR fixes any broken links and updates versions in CSI Unity XT and CSM Operator for Powerstore driver

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/658 |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/_index.md [hunk 1]
```diff
@@ -17,16 +17,12 @@ The CSI Drivers by Dell implement an interface between [CSI](https://kubernetes-
 |               | PowerMax         | PowerFlex           | Unity XT         | PowerScale        | PowerStore       |
 |---------------|:----------------:|:-------------------:|:----------------:|:-----------------:|:----------------:|
 | Kubernetes    | 1.24, 1.25, 1.26 | 1.24, 1.25, 1.26    | 1.24, 1.25, 1.26 | 1.23, 1.24, 1.25  | 1.24, 1.25, 1.26 |
-<<<<<<< HEAD
-| RHEL          |     7.x,8.x      |     7.9,8.5,8.6     |     7.x,8.x      |     7.x,8.x       |     7.9,8.6      |
-=======
-| RHEL          |     7.x,8.x      |     7.x,8.x         |     7.x,8.x      |     7.x,8.x       |     7.9,8.6      |
```
[the remark this answers is not in the document]
Comment (panigs7) [reply content/docs/csidriver/_index.md:23]: @rajkumar-palani This is already present in the latest release-1.6.0 branch . I am not able to do a git rebase --continue as I havent started any rebase. Also, I have kept only one instance of the content.

File: content/docs/deployment/csmoperator/drivers/powerstore.md [hunk 2]
```diff
@@ -71,15 +71,15 @@ User can query for all Dell CSI drivers using the following command:
 
   | Parameter | Description | Required | Default |
 | --------- | ----------- | -------- |-------- |
-| replicas | Controls the number of controller pods you deploy. If the number of controller pods is greater than the number of available nodes, the excess pods will be pending state till new nodes are available for scheduling. Default is 2 which allows for Controller high availability. | Yes | 2 |
-| namespace | Specifies namespace where the drive will be installed | Yes | "test-powerstore" |
-| fsGroupPolicy | Defines which FS Group policy mode to be used, Supported modes `None, File and ReadWriteOnceWithFSType` | No |"ReadWriteOnceWithFSType"|
-| storageCapacity | Enable/Disable storage capacity tracking feature | No | true |
+| replicas | Controls the number of controller pods you deploy. If the number of controller pods is greater than the number of available nodes, the excess pods will be in pending state till new nodes are available for scheduling. Default is 2 which allows for Controller high availability. | Yes | 2 |
```
Comment (rsedlock1958) [inline content/docs/deployment/csmoperator/drivers/powerstore.md:74]: Change till to until.

**Review by rsedlock1958 — COMMENTED** [review]
1 minor edit required.

File: content/docs/deployment/csmoperator/drivers/powerstore.md (hunk 2 above)
Comment (panigs7) [reply content/docs/deployment/csmoperator/drivers/powerstore.md]: Done

**Review by rensyct — DISMISSED** [review]

**Review by rajendraindukuri — DISMISSED** [review]

File: content/docs/csidriver/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (panigs7) [reply content/docs/csidriver/_index.md]: Rebased with the latest merge to release-1.6.0

[the remark this answers is not in the document]
Comment (rajkumar-palani) [reply content/docs/csidriver/_index.md]: Thanks @panigs7

**Review by rajkumar-palani — APPROVED** [review]
lgtm

**Review by rensyct — APPROVED** [review]

**Review by rajendraindukuri — APPROVED** [review]

**Pull request merged**